Analysis/interface/TauIdResults.h

```diff
@@ -57,6 +57,8 @@ struct TauIdResults {
         "VVLoose VLoose Loose Medium Tight VTight VVTight") \
     TAU_ID(byIsolationMVArun2017v2DBoldDMdR0p3wLT2017, "by{wp}IsolationMVArun2017v2DBoldDMdR0p3wLT{raw}2017", true, \
         "VVLoose VLoose Loose Medium Tight VTight VVTight") \
+    TAU_ID(byIsolationMVADBnewDMwLTPhase2, "by{wp}IsolationMVADBnewDMwLTPhase2{raw}", true, \
+        "VVLoose VLoose Loose Medium Tight VTight VVTight") \
```

Comment:
[FillTuple](https://github.com/cms-tau-pog/TauMLTools/blob/master/Analysis/interface/TauIdResults.h#L119-L134) should be modified to fill the raw and id branches with default values if ID not found. Otherwise, the code will not work for both Phase 2 and Run 2 samples.

Modified version (possibly some renaming will be needed) can be copied from [here](https://github.com/hh-italian-group/h-tautau/blob/master/Core/include/TauIdResults.h#L65-L82)

Reply:
Done & tested under `CMSSW_10_6_15` for DYJets MC